### ISSUE TITLE
Add pause mechanism to background animation.

### ISSUE THREAD

**Issue by andrewmallis** [kalamuna/kalastatic#188]
in #30 we developed a fun homepage, but we should be sure that we meet WCAG 2.0

> Some people with seizure disorders can have a seizure triggered by flashing visual content. Most people are unaware that they have this disorder until it strikes. In 1997, a cartoon on television in Japan sent over 700 children to the hospital, including about 500 who had seizures.

To remedy, I think we need to ensure we are not flashing more than three times in any one second period.

Additionally, we need to provide a mechanism that stops the iteration

> For any moving, blinking or scrolling information that (1) starts automatically, (2) lasts more than five seconds, and (3) is presented in parallel with other content, there is a mechanism for the user to pause, stop, or hide it
## Reference

http://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-pause.html
http://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure.html
http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/G19

**Comment by andrewmallis** [comment]
tab focus or mouse hover over the area = pause the playback?

**Comment by soniktrooth** [comment]
I don't know that you can 'pause' a gif, but with js we could swap to a static image.

**Comment by soniktrooth** [comment]
This seems prescient http://alistapart.com/article/designing-safer-web-animation-for-motion-sensitivity

**Comment by andrewmallis** [comment]
Looks great on DEV.
With the frame rate slowed down so much, we could think about only showing one loop.

I am going to put this issue in the backlog again, because although this is better overall for the UX, the inability to pause the looping still has us violate web accessibility standards. It's not super flagrant at this point.

**Comment by RobLoach** [comment]
This is really old, and not an issue in KSTAT 2. Only pertains to the KSTAT prototype example. Suggest closing this.

**Comment by andrewmallis** [comment]
This is a larger design pattern issue, and affects a particular component of the kstat demo website. Perhaps we should have a tag specific to our demo/website/blueprint work?
Returning to ICEBOX till we can chart a course of action.
